docs/en/metadata_oidc_rp.rst

```diff
@@ -59,6 +59,12 @@ The RP Metadata of type **"openid_relying_party"** MUST contain at least the fol
   * - **id_token_signed_response_alg**
     - See `OpenID.Registration#ClientMetadata`_.
     - |spid-icon| |cieid-icon| 
+  * - **id_token_encrypted_response_alg**
+    - See `OpenID.Registration#ClientMetadata`_. This is optional. If this is requested, the response will be signed then encrypted, with the result being a Nested JWT, as defined in :rfc:`7515#section-4`.
```

Comment:
Do we have MUST be or "will be" as defined in https://openid.net/specs/openid-connect-registration-1_0.html

Reply:
fixed in https://github.com/italia/spid-cie-oidc-docs/pull/55/commits/8ec33eca96a7075ce9c222eb16e07b2fc6820176